cmd/up/up.go

```diff
@@ -247,7 +250,7 @@ func (up *upContext) start(autoDeploy, build bool) error {
 	stop := make(chan os.Signal, 1)
 	signal.Notify(stop, os.Interrupt)
 
-	analytics.TrackUp(true, up.Dev.Name, up.getInteractive(), len(up.Dev.Services) == 0, up.isSwap, up.Dev.RemoteModeEnabled())
+	analytics.TrackUp(true, up.Dev.Name, up.getInteractive(), len(up.Dev.Services) == 0, up.isSwap, up.Dev.Divert != nil)
```

Comment:
could we have a separate event for divert?

Reply:
it is just another field in the okteto manifest to configure the `up` behavior. it makes it simple to compare the usage of divert against other `up` executions